applications/wg-easy/Taskfile.yaml

```diff
@@ -172,6 +172,36 @@ tasks:
     deps:
       - cluster-create
 
+  helm-preflight-dryrun:
```

Comment:
Do we really need a separate command just for a dry-run? This can't be combined with the helm-preflight command with an argument of some sort?

Reply:
Great idea, I've removed this task and update `helm-preflight` with `DRY_RUN` argument, we can now run

```
task helm-preflight
task helm-preflight DRY_RUN=true
```